database_schema.md

```diff
@@ -136,6 +136,76 @@ seat_id: {
 }
 ```
 
+### Minutes Item
+```
+minutes_item: {
+    minutes_item_file_id: str
+    minutes_item_id: str
+    name: str
+    uri: str
```

Comment:
What is `uri` used for? Is this the `uri` of `minutes_item_file_id`?

Reply:
Yeah I should've clarified that, but I'm gonna remove it since there can be multiple `minutes_item_file` for a `minutes_item`.